translations/en_GB/atrium-translations-en_GB-common/src/main/kotlin/ch/tutteli/atrium/translations/DescriptionIterableAssertion.kt

```diff
@@ -46,7 +46,9 @@ enum class DescriptionIterableAssertion(override val value: String) : StringBase
     NEXT_ELEMENT("a next element"),
     NO_ELEMENTS("❗❗ cannot be determined, empty Iterable"),
     DUPLICATE_ELEMENTS("duplicate elements"),
-    DUPLICATED_BY("duplicated by index: %s")
+    DUPLICATED_BY("duplicated by index: %s"),
+    ENTRY_NOT_FOUND("but no such entry was found."),
+    VALUE_NOT_FOUND("but no such item was found.")
```

Comment:
Or potentially `"but no such value was found"` or `"but no such values were found"`

Reply:
It's always a good idea to write `I'll work on this` in an issue because this way I have a chance to update the description in case it is outdated (it would take me too much time to do this pro-actively) which was unfortunately the case for this issue. I have updated the description, it should have been `but no such element was found` which I would use for both ENTRY_NOT_FOUND and VALUE_NOT_FOUND. The report then looks as follows:
```
expected that subject: kotlin.sequences.SequencesKt___SequencesKt$asIterable$$inlined$Iterable$1@34ace742        (kotlin.sequences.SequencesKt___SequencesKt$asIterable$$inlined$Iterable$1 <883746626>)
◆ contains, in any order: 
  ⚬ an element which equals: 9.5        (kotlin.Double <7392225>)
      » but no such element was found
```

Reply:
Got it, I'll keep this in mind for the future.